packages/react/package.json

```diff
@@ -4,7 +4,10 @@
   "description": "React tools for MUD client.",
   "license": "MIT",
   "source": "src/index.ts",
-  "main": "src/index.ts",
+  "type": "module",
+  "types": "dist/index.d.ts",
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
```

Comment:
The way this repo in general handles setting all this up is a bit of a mess.  This is what build tools use to find the types, esm, or cjs.   I would clean this up everywhere.

Reply:
Currently we're pointing `main` to the typescript source in order to have instant updates during development, and then only set it to `dist/index.js` in the `prerelease` step. Down to change this if find an equally performant alternative to pointing to the ts source for development though